Source/Core/Common/MemoryUtil.cpp

```diff
@@ -79,25 +146,24 @@ void* AllocateExecutableMemory(size_t size, bool low)
   {
     ptr = nullptr;
 #endif
-    PanicAlert("Failed to allocate executable memory. If you are running Dolphin in Valgrind, try "
-               "'#undef MAP_32BIT'.");
+    PanicAlert("Failed to allocate executable memory.");
   }
-#if !defined(_WIN32) && defined(_M_X86_64) && !defined(MAP_32BIT)
+#if !defined(_WIN32) && defined(_M_X86_64)
   else
   {
-    if (low)
+    if (map_hint <= 0xFFFFFFFF)
```

Comment:
Should this one have an `UL` suffix as well? And: same named constant as above, if you decide to introduce one?